### ISSUE TITLE
setup.py needs to specify an encoding when opening README.md

### ISSUE THREAD

**Issue by anntzer** [rflamary/POT#33]
README.md contains non-ascii characters, so setup.py will fail if the locale is ascii, e.g.
```
$ LC_ALL=C python setup.py install
Traceback (most recent call last):
  File "setup.py", line 26, in <module>
    import pypandoc
ModuleNotFoundError: No module named 'pypandoc'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "setup.py", line 29, in <module>
    README = open(os.path.join(ROOT, 'README.md')).read()
  File "/usr/lib/python3.6/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 5501: ordinal not in range(128)
```

This is easily fixed by using `open(..., encoding="utf-8")` (or, if you want Py2 compatibility, `codecs.open(..., encoding="utf-8")`).

**Comment by ncourty** [comment]
Thanks @anntzer for reporting this issue and the patch. I updated accordingly setup.py
Fixes #33

**Comment by anntzer** [comment]
As a side note https://packaging.python.org/specifications/#description-content-type (https://setuptools.readthedocs.io/en/latest/history.html#v36-4-0) could be useful wrt your use of pypandoc (but that's not really an issue for me).
